### DISCUSSION TITLE
MIDI2LR not detecting midi device.

### DISCUSSION THREAD

**Discussion by loybuckz** [rsjaffe/MIDI2LR#731] [Problems]
Hi! I haven't used Midi2LR for more than 2 years due to the pandemic (no clients). I have installed the software today and MIDI2LR is not recognizing the inputs from my MIDI controller. My setup is as follows:
Operating System: Windows 11
LR: Lightroom Classic 2023 12.0
Controller: Behringer X-Touch Mini

**Comment by antzpantz** [comment]
Try deleting C:\Users\<your username>\AppData\Roaming\MIDI2LR
and C:\Users\<your username>\AppData\Roaming\Adobe\Lightroom\Modules\MIDI2LR.lrplugin
and reinstall.

Make sure you backup/export your current settings and button configs before you uninstall.

**Reply by loybuckz** [reply]
That worked! Thank you!

**Comment by DGrove88** [comment]
Same problem doing what Antzpantz suggested did not fix it. Does the creator of Midi2LR have to create a profile for every controller ahead of time to work? I got a not well known brand and suspect that's why. It's a Avatar emp16.
